samples/Mvc.FileUpload/Controllers/FileController.cs

```diff
@@ -0,0 +1,174 @@
+        {
+            if (!MultipartRequestHelper.IsMultipartContentType(Request.ContentType))
+            {
+                return BadRequest($"Expected a multipart request, but got '{Request.ContentType}'.");
+            }
+
+            // Used to accumulate all the form url encoded key value pairs in the request.
+            var formAccumulator = new KeyValueAccumulator();
+            string targetFilePath = null;
+
+            var boundary = MultipartRequestHelper.GetBoundary(
+                MediaTypeHeaderValue.Parse(Request.ContentType), _defaultFormOptions.MultipartBoundaryLengthLimit);
+            var reader = new MultipartReader(boundary, HttpContext.Request.Body);
+
+            var section = await reader.ReadNextSectionAsync();
+            while (section != null)
+            {
+                ContentDispositionHeaderValue contentDisposition;
+                ContentDispositionHeaderValue.TryParse(section.ContentDisposition, out contentDisposition);
+
+                if (MultipartRequestHelper.HasFileContentDisposition(contentDisposition))
+                {
+                    var name = HeaderUtilities.RemoveQuotes(contentDisposition.Name) ?? string.Empty;
+                    var fileName = HeaderUtilities.RemoveQuotes(contentDisposition.FileName) ?? string.Empty;
+
+                    // Copy the uploaded file to local disk
+                    targetFilePath = Path.Combine(_hostingEnvironment.ContentRootPath, Guid.NewGuid().ToString());
+                    using (var targetStream = System.IO.File.Create(targetFilePath))
+                    {
+                        await section.Body.CopyToAsync(targetStream);
+
+                        _logger.LogInformation($"Copied the uploaded file '{fileName}' to '{targetFilePath}'.");
+                    }
+
+                    //// NOTE: Following is an example of copying the uplaoded file's raw stream to an Azure blob
+                    //// Note that Azure development storage account is being used here
+                    //var cloudStorageAccount = CloudStorageAccount.Parse(connectionString: "UseDevelopmentStorage=true");
+                    //var cloudBlobClient = cloudStorageAccount.CreateCloudBlobClient();
+
+                    //// NOTE: container name MUST be lower case
+                    //var cloudBlobContainer = cloudBlobClient.GetContainerReference("mvcfileuploadcontainer");
+                    //await cloudBlobContainer.CreateIfNotExistsAsync();
+
+                    //var blobId = Guid.NewGuid().ToString();
+                    //var cloudBlockBlob = cloudBlobContainer.GetBlockBlobReference(blobId);
+
+                    //// Copy the uploaded file's raw stream directly to the blob
+                    //await cloudBlockBlob.UploadFromStreamAsync(section.Body);
+
+                    //targetFilePath = blobId;
+                    //_logger.LogInformation($"Copied the uploaded file '{fileName}' to blob with id '{blobId}'.");
```

Comment:
I don't think there's any value to having this code here. This just bloats the sample - if we need to provide a sample for blob storage let's do that separately.